src/gsy_e/gsy_e_core/user_profile_handler.py

```diff
@@ -354,7 +356,7 @@ def get_profile_type(self, profile_uuid: str) -> InputProfileTypes:
         """Read the profile type from a profile with the specified UUID."""
         return self.db.get_profile_type_from_db_buffer(profile_uuid)
 
-    def _get_profile_uuids_from_setup(self, area: "Area", profile_uuids: List) -> None:
+    def _get_profile_uuids_from_setup(self, area: "Area", profile_uuids: List) -> List:
```

Comment:
Please just try it out and it will most probably pass you unit tests. Thanks a lot

Reply:
I understand now, that it works because of the `extend()`.
I did that here: [717b796](https://github.com/gridsingularity/gsy-e/pull/1550/commits/717b796f782684fd7c2e18ec9bfda8f386518586)

Reply:
Thanks a lot